pkgs/games/asphyxia-core/default.nix

```diff
@@ -0,0 +1,75 @@
+with import <nixpkgs> { };
+
+stdenv.mkDerivation rec {
+  pname = "asphyxia-core";
+  version = "1.40";
+
+  src = fetchurl {
+    url = "https://github.com/asphyxia-core/asphyxia-core.github.io/releases/download/v${version}/asphyxia-core-linux-x64.zip";
+    sha256 = "sha256-YFAXHzxoLuKfflwqYvpifefRE2/KbShuQ0v+0LG7OCg=";
+  };
+
+  dontStrip = true;
```

Comment:
Why?

Reply:
this is required for the mystical hack down below - will move & add a comment.